src/test/groovy/ratpack/zipkin/ServerTracingModuleSpec.groovy

```diff
@@ -368,4 +371,49 @@ class ServerTracingModuleSpec extends Specification {
 		then:
 			assertThat(reporter.getSpans()).isNotEmpty()
 	}
+
+	def 'Should nest client span and server span should have the same trace id'() {
+		given:
+			def webServer = new MockWebServer()
+			webServer.start()
+			webServer.enqueue(new MockResponse().setResponseCode(200))
+			def url = webServer.url("/")
```

Comment:
@llinder ah ic...forgot that you'd done it like that. I'll refactor the client tests in a separate PR.